include/odp/api/spec/packet.h

```diff
@@ -1711,6 +1711,9 @@ odp_packet_chksum_status_t odp_packet_l3_chksum_status(odp_packet_t pkt);
  * attempt. It depends on packet input (or IPSEC) configuration, packet content
  * and implementation capabilities if checksum check is attempted for a packet.
  *
+ * Checksum check result for an UDP packet with checksum field value of zero
```

Comment:
Might be worth clarifying that this is for IPv4 UDP packets with zero checksums?